### ISSUE TITLE
Publish Data or Not? Re: Specimens from us and now in other collections -Georeferences

### ISSUE THREAD

**Issue by bradmROM** [tdwg/dwc-qa#78]
Please refer to the issue noted here: https://github.com/tdwg/dwc-qa/issues/77

If I publish them should I update locality information to add Georeferences for each record even though they now reside in other collections etc. Most of these specimens have had the localities georeferenced during ORNIS georeferencing and the data is already within the collection records. I would be simple to update them before publishing.

Or should I assume that these records already have these particular fields published with data.

Brad Millen, ROM, Toronto, Canada

**Comment by tucotuco** [comment]
Hi Brad,

I think this is a good opportunity for community building, in the way we saw in the early vertebrate networks (MaNIS, HerpNET, ORNIS) when we did collaborative georeferencing. Ideally, the collection acting as permanent custodian of the specimen should be the one to keep the definitive record of it, including any georeference or other information, wherever that information might come from. If the collection no longer exists, or if it isn't digitized, these might be good candidates for an exception, and publishing those data with a clear expression of the disposition of the specimen would ensure that the data, at least, are not lost to posterity.

There are some interesting cases, though, where things aren't so simple. Botanical duplicates and other individual organisms with material samples in more than one location are a case in point. This situation is common in botany, and informaticists are keen to provide communication mechanisms between the collections housing the different parts so that they can each benefit by new taxonomic determinations and the like.
